sarahbenmessaoud.svg

```diff
@@ -0,0 +1,106 @@
+<svg viewBox="0 -25 400 200" xmlns="http://www.w3.org/2000/svg">
+  <!--Update the title with your username-->
+  <title>@sarahbenmessaoud</title>
+<g stroke="black" stroke-width="2">
+<rect x="3" y="-20" width="40" height="40" fill="yellow" />
+<circle cx="100" cy="100" r="40" fill="blue" />
+<polygon points="120,63.3 90,106.7 45,106.7 20,63.3 45,20 95,20" fill="thistle"/>
+<line x1="1" y1="200" x2="0" y2="0" stroke-width="20" stroke=""/>
+<path
+  d="M 95 20 L 60 20 L 60 100 z" fill="bisque"
+/>
+  <text x="150" y="115" fill="blue" stroke="none" font-size="40" font-weight="700">ART
+    <tspan x="250" y="45" fill="red" font-size="14">What is the purpose of</tspan></text>/>
+     </g>
+</svg>
+
+
+<!-- NOTHING TO SEE HERE... MOVE ALONG!!! -->
```

Comment:
Three more thing!

1. Remove the below `<script>` (and its contents)
2. Add a `font-family` on any element
3. Add a `opacity` attribute on any element

Reply:
Oeeh wist niet dat de laatste twee verplicht waren, maar heb ze toegevoegd!